docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+:::note
+Optionally, you can verify the signatures of the [helm charts](https://docs.kubewarden.io/security/verifying-kubewarden#helm-charts) and [container images](https://docs.kubewarden.io/security/verifying-kubewarden#container-images)
+:::
+2. Add `cert-manager` if it is not available in your private registry. 
+```
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm pull jetstack/cert-manager --version v1.7.1
+helm template ./cert-manager-v1.7.1.tgz | awk '$1 ~ /image:/ {print $2}' | sed s/\"//g >> ./kubewarden-images.txt
+```
+3. Download [kubewarden-save-images.sh](https://github.com/kubewarden/utils/blob/main/scripts/kubewarden-save-images.sh) 
+and [kubewarden-load-images.sh](https://github.com/kubewarden/utils/blob/main/scripts/kubewarden-load-images.sh) from the Kubewarden utils repository.
+4. Save Kubewarden container images into a tar.gz file:
+```
+./kubewarden-save-images.sh --image-list ./kubewarden-images.txt --images kubewarden-images.tar.gz
+```
+Docker begins pulling the images used for an airgap install. Be patient. This process takes a few minutes. 
+When the process completes, your current directory will output a tarball named `kubewarden-images.tar.gz`. Check that the output is in the directory.
+
+## Save policies in your workstation
+
+[kwctl](https://github.com/kubewarden/kwctl/releases) version v1.3.1 or above is required for this step.
+
+1. Add all the policies you want to use in a `policies.txt` file. 
+Example: (TODO add link to default policies.txt once it is available) 
+```
+registry://ghcr.io/kubewarden/policies/pod-privileged:v0.2.2
+registry://ghcr.io/kubewarden/policies/safe-labels:v0.1.5
+```
+2. Download `kubewarden-save-policies.sh` and `kubewarden-load-policies.sh` from the [kwctl repository](https://github.com/kubewarden/kwctl/tree/main/scripts)
+3. Save policies into a tar.gz file:
+```
+./kubewarden-save-policies.sh --policies-list policies.txt
+```
+kwctl downloads all the policies and stores then in `kubewarden-policies.tar.gz`
+
+## Helm charts
+
+You need to download the following helm charts in your workstation:
+
+```
+helm pull kubewarden/kubewarden-crds         
+helm pull kubewarden/kubewarden-controller
+helm pull kubewarden/kubewarden-defaults  
+helm pull jetstack/cert-manager         
+```
+
+Then move the helm chart tar.gz files to your air-gapped environment.
+
+## Populate private registry
+
+Move `kubewarden-policies.tar.gz`, `kubewarden-images.tar.gz`, `kubewarden-load-images.sh`, `kubewarden-load-policies.sh` and `policies.txt`
+to the air-gapped environment.
+
+1. load Kubewarden images into the private registry
+```
+./kubewarden-load-images.sh --image-list ./kubewarden-images.txt --images kubewarden-images.tar.gz --registry <REGISTRY.YOURDOMAIN.COM:PORT>
```

Comment:
Point out that the docker client must be authenticated against the local registry. Explain where all the images are going to be pushed (like `<REGISTRY.YOURDOMAIN.COM:PORT>/kubewarden`. That's because some users might has internal IT to create the `kubewarden` namespace and give them push and pull rights to it.

Reply:
https://github.com/kubewarden/docs/pull/150/commits/618c4564566ba988e8259624178384849f7c5d7a